src/schedule.jl

```diff
@@ -346,3 +358,12 @@ function today(::Val{:text}; now, track, terminal_links)
     push!(strings, legend)
     return strings
 end
+
+function tomorrow(;
+    now=default_now(),
+    track=nothing,
+    terminal_links=TERMINAL_LINKS,
+    output=:terminal, # can take the :text value to output a Vector{String}
+)
+    return today(Val(output); now = now + Dates.Day(1), track, terminal_links, highlighting = false)
```

Comment:
The `highlighting = false` option should not just hide the legend entry but also make sure that nothing gets highlighted (e.g. in `_get_current_talk_highlighter`). AFAICS, since you are changing the `now`, there will be entries (falsely) highlighted in yellow in the `tomorrow()` output. Or am I missing something?